WordPress/Classes/Utility/ABTesting/ExPlatService.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+class ExPlatService {
+    let wordPressComRestApi: WordPressComRestApi
+
+    let assignmentsPath = "wpcom/v2/experiments/0.1.0/assignments/calypso"
+
+    init(wordPressComRestApi: WordPressComRestApi) {
+        self.wordPressComRestApi = wordPressComRestApi
+    }
+
+    func getAssignments(completion: @escaping (Assignments?) -> Void) {
+        wordPressComRestApi.GET(assignmentsPath,
+                                parameters: nil,
+                                success: { responseObject, httpResponse in
```

Comment:
what about this since you're not using it? 

```suggestion
                                success: { responseObject, _ in
```